src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -432,8 +432,21 @@ class BaseOptionsSelector extends Component {
                 isRowMultilineSupported={this.props.isRowMultilineSupported}
                 isLoadingNewOptions={this.props.isLoadingNewOptions}
                 shouldPreventDefaultFocusOnSelectRow={this.props.shouldPreventDefaultFocusOnSelectRow}
+                nestedScrollEnabled={this.props.nestedScrollEnabled}
+                bounces={this.props.bounces}
```

Comment:
I think `bounces` prop is redundant.
```suggestion
                bounces={!this.props.shouldTextInputAppearBelowOptions || !this.props.shouldAllowScrollingChildren}
```